### PR TITLE
Support multiple disks in a VM

### PR DESCRIPTION
**Pull request by koln67**
Allow multiple disks to be added in the create VM step

### CODE REVIEW

**Comment by koln67** [comment]
Currently unsure about AttachedDisk and Disk, so pr is on hold

File: imagetest/testworkflow.go [hunk 1]
```diff
@@ -69,7 +80,13 @@ func (t *TestWorkflow) appendCreateVMStep(name, hostname string) (*daisy.Step, *
 	instance.StartupScript = fmt.Sprintf("wrapper%s", suffix)
 	instance.Name = name
 	instance.Scopes = append(instance.Scopes, "https://www.googleapis.com/auth/devstorage.read_write")
-	instance.Disks = append(instance.Disks, attachedDisk)
+
+	for _, disk := range disks {
```
Comment (dorileo) [inline imagetest/testworkflow.go:84]: are you sure we want do add all items in the disks slice? I guess we want to skip `root` disk (AKA `disks[0]`), right?

Comment (koln67) [reply imagetest/testworkflow.go:84]: before this PR, there was only one item passed in as the disk, and that disk was appended.

I'm assuming that should be added, however I'm trying to figure out if it should be of type "Disk" or "AttachedDisk."

It is also possible that the first disk should be of type "AttachedDisk" and the second and future disks should be of type "Disk"

Comment (koln67) [reply imagetest/testworkflow.go:84]: It appears that instance.Disks starts out as a slice with length 0. This means we want to append all the disks.

**Comment by koln67** [comment]
/hold

It seems that using a map from vm name to Attached Disk Params might be easier

File: imagetest/testworkflow.go (hunk 1 above)
Comment (dorileo) [reply imagetest/testworkflow.go]: sounds good.

**Pull request merged**